### ISSUE TITLE
"Switch headers to text" button is not clearly visible in Aquatic, Dusk & Night sky mode.

### ISSUE THREAD

**Issue by shahparth-lab** [Azure/LogicAppsUX#3260]
### Describe the Bug with repro steps 

User Experience: 

 Low vision user will be impacted if controls will not be clearly visible in high contrast modes. Due to this issue, users will be unable to see the content in the proper way.

Note: 

 User credentials should NOT be included in the bug.

Repro Steps:

 Open [https://ms.portal.azure.com/](<https://ms.portal.azure.com/>) and login via valid credentials.

 Navigate to the Left Nav bar and select 'More Services', search 'Logic Apps' and select it. Select any already created logic app

 Select 'Logic App Designer' in left navigation pane.

 Go to Templates and Select any template like 'HTTP Request Response'.

 Navigate all the controls on the 'HTTP Request Response' pop up. Press 'Use this template' button. Press Save Button.

 Make some changes to the workflow and Press Discard button. Navigate all the controls on the dialog box that appears.

 Navigate all the controls on the 'Logic App Designer' blade like top menu bar (Save, Discard, Run, Designer, Code View, Templates, Connectors, Hep), Designer section below the menu …etc.

 Verify the issue.

Actual Result:

 The "Switch headers to text" button present under the 'Response' section is not clearly visible in high contrast Aquatic, Dusk & Night sky modes.

Expected Result:

 The "Switch headers to text" button present under the 'Response' section should be clearly visible in high contrast Aquatic, Dusk & Night sky modes.

### What type of Logic App Is this happening in? 

Consumption (Portal)

### Are you using new designer or old designer 

New Designer

### Did you refer to the TSG before filing this issue? [https://aka.ms/lauxtsg](<https://aka.ms/lauxtsg>) 

Yes

### Workflow JSON 

*No response*

### Screenshots or Videos 

*No response*

### Browser 

OS: Windows 11 Version 21H2 (OS build: 22000.795)

 Browser: Chrome Version 103.0.5060.134 (Official Build) (64-bit)

### Additional context 

*No response*

AB#25072821
